tasks_downstream.py

```diff
@@ -700,11 +695,9 @@ def test(
 )
 def stop(c, purge=False):
     """Stop and (optionally) purge environment."""
-    cmd = "docker-compose"
+    cmd = "docker-compose down"
```

Comment:
The resources you're cleaning up are meaningless.

Networks that are not attached consume no IP namespaces, and containers consume almost no disk space. The meaningful things are images and volumes. Besides, when pruning volumes, you may end up removing volumes that you wouldn't want to remove, because they'd be considered garbage by Docker because they'd be attached to no containers.

And stopping/starting the environment would be slower.

You'll have to do the `docker system prune` every now and then anyways due to dangling volumes and images, so what's the benefit of this? 🤔

Reply:
Well, my main goal was to avoid the IP saturation caused by having so many networks...

> Networks that are not attached consume no IP namespaces

I don't know if this is true... Maybe I have something wrong with my config, but from time to time I get the error saying that docker can't find a non-overlapping IP. The networks do have containers attached, so they must be keeping some IP reference... Can you point some documentation to see if I'm doing something wrong?

That was the main benefit, and I think the slightly bigger stop/start times wouldn't be that meaningful, as oppose to the time spent each time you have to prune the networks, plus running a `docker-compose down` in each project to recreate the networks that no longer exist but that have containers associated...